src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(word -> word.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
use one mapping whenever its possible 
```suggestion
                .flatMap(str -> Arrays.stream(str.split(",")))
```